src/pe_source/data/helpers/redact_pii.py

```diff
@@ -0,0 +1,560 @@
+all_cards = r"\b((4\d{3}|5[1-5]\d{2}|2\d{3}|3[47]\d{1,2})[\s\-]?\d{4,6}[\s\-]?\d{4,6}?([\s\-]\d{3,4})?(\d{3})?)\b"
+US_phones = r"((\+|\b)[1l][\-\. ])?\(?\b[\dOlZSB]{3,5}([\-\. ]|\) ?)[\dOlZSB]{3}[\-\. ][\dOlZSB]{4}\b"
+US_street_address = r"\d{1,8}\b[\s\S]{10,100}?\b(AK|AL|AR|AZ|CA|CO|CT|DC|DE|FL|GA|HI|IA|ID|IL|IN|KS|KY|LA|MA|MD|ME|MI|MN|MO|MS|MT|NC|ND|NE|NH|NJ|NM|NV|NY|OH|OK|OR|PA|RI|SC|SD|TN|TX|UT|VA|VT|WA|WI|WV|WY)\b\s\d{5}"
+
+
+def redact_pii(df, column_list=[]):
+    """Run through provided columns and redact PII."""
+    if column_list:
+        for column in column_list:
+            df = scrub(df, column)
+            df[column] = df[column].replace(
+                regex={
+                    all_cards: "{{CREDIT_CARD}}",
+                    US_street_address: "{{ADDRESS}}",
+                    email: "{{EMAIL}}",
+                }
+            )
+    else:
+        for column in df.columns:
+            df = scrub(df, column)
+        df = df.replace(
+            regex={
+                all_cards: "{{CREDIT_CARD}}",
+                US_street_address: "{{ADDRESS}}",
+                email: "{{EMAIL}}",
+            }
+        )
+    return df
+
+
+def scrub(df, column):
+    """Add different scrubber classes and run column through scrubadub."""
+    scrubber = scrubadub.Scrubber()
+    scrubber.remove_detector("url")
+    scrubber.remove_detector("twitter")
+    scrubber.remove_detector("email")
+    scrubber.add_detector(SSNDetector)
+    scrubber.add_detector(PassportDetector)
+    scrubber.add_detector(AlienIdDetector)
+    scrubber.add_detector(FL_DLDetector)
+    scrubber.add_detector(HI_NE_VA_DLDetector)
+    scrubber.add_detector(IL_DLDetector)
+    scrubber.add_detector(MN_FL_MD_MI_DLDetector)
+    scrubber.add_detector(MO_OK_DLDetector)
+    scrubber.add_detector(MD_DLDetector)
+    scrubber.add_detector(CA_DLDetector)
+    scrubber.add_detector(CO_DLDetector)
+    scrubber.add_detector(ID_DLDetector)
+    scrubber.add_detector(NJ_DLDetector)
+    scrubber.add_detector(NY_DLDetector)
+    scrubber.add_detector(ND_DLDetector)
+    scrubber.add_detector(OH_DLDetector)
+    scrubber.add_detector(PA_DLDetector)
+    scrubber.add_detector(VT_DLDetector)
+    scrubber.add_detector(VA_DLDetector)
+    scrubber.add_detector(WA_DLDetector)
+    scrubber.add_detector(WV_DLDetector)
+    scrubber.add_detector(WI_DLDetector)
+    scrubber.add_detector(WY_DLDetector)
+    scrubber.add_detector(NH_DLDetector)
+    scrubber.add_detector(IO_DLDetector)
+    scrubber.add_detector(IA_DLDetector)
+    scrubber.add_detector(KS_DLDetector)
+    scrubber.add_detector(KY_DLDetector)
+    scrubber.add_detector(MI_DLDetector)
+    df[column] = df[column].apply(lambda x: scrubber.clean(x))
+
+    analyzer = AnalyzerEngine()
+    anonymizer = AnonymizerEngine()
+    entities = [
+        "PHONE_NUMBER",
+        "CREDIT_CARD",
+        "US_DRIVER_LICENSE",
+        "US_SSN",
+        "EMAIL_ADDRESS",
+        "IP_ADDRESS",
```

Comment:
Can this list be alphabetized without breaking functionality?
```suggestion
        "CREDIT_CARD",
        "EMAIL_ADDRESS",
        "IP_ADDRESS",
        "PHONE_NUMBER",
        "US_DRIVER_LICENSE",
        "US_SSN",
```